### ISSUE TITLE
install_requires in setup.py invalid

### ISSUE THREAD

**Issue by Ultimation** [python/mypy_extensions#9]
Currently it state it requires a python version 3.5 or lower

It also currently fails running:
```
python3 setup.py
error in mypy_extensions setup command: 'install_requires' must be a string or list of strings containing valid project/version requirement specifiers; Expected ',' or end-of-list in typing >= 3.5.3; python_version < "3.5" at ; python_version < "3.5"
```

**Comment by JelleZijlstra** [comment]
No, it currently requires `typing` as a dependency on Python <3.5 only.

Why are you using `setup.py` directly? Generally this module should be installed through pip. Perhaps you are running an old version of setuptools.

**Comment by Ultimation** [comment]
We're building this package as an RPM for our deployment methods. We don't typically run setup.py directly but I did to emphasis that it did not like the syntax. 

However - I see what you mean: https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-platform-specific-dependencies

It looks like the latest centos setuptools does not support this syntax. 
Mypy doesn't work on centos anyway as it's python is 3.4. I agree this is fine on a latest fedora server - sorry.

This was during the debug into other issues with building the package, typically we run the build commands inside a mock environment, but I ran it directly in this case.
